bbz-project/src/components/RecordsListComponents/RecordComps/Record.js

```diff
@@ -4,13 +4,33 @@ import Sublist from "./Sublist";
 import RecordOptions from "./RecordOptions";
 import { useRecordAnimations } from "../../../animations/useRecordAnimations";
 import { animated } from "react-spring";
-import {
-  FIELD_WIDTHS,
-  FIELD_WIDTHS_SHORT,
-} from "../../../constants/records-list-manager-const";
+import LinesEllipsis from "react-lines-ellipsis";
 
-const Record = ({ data, keysAmount }) => {
-  const keys = Object.keys(data);
+const Record = ({
+  publicationDate,
+  bookAuthor,
+  title,
+  publisher,
+  publicationPlace,
+  source,
+  id,
+  subtitle,
+  originalEdition,
+  pages,
+  language,
+  series,
+  isbnIssnNumber,
+  keywordsAndContent,
```

Comment:
I think there are too many props, it makes the code unnecessarily too long. Maybe make an object where all of these props are members: e.g. recordData.publicationDate, recordData.id, etc.